module/VuFind/src/VuFind/Controller/Feature/RecordVersionsSearchTrait.php

```diff
@@ -0,0 +1,101 @@
+<?php
+/**
+ * VuFind Action Feature Trait - Record Versions Search
```

Comment:
Might be worth documenting the dependencies of this trait in a comment somewhere.

Reply:
(e.g. https://github.com/vufind-org/vufind/blob/dc4677e100ecd786e5bb4b5b434a6ea08025aa9c/module/VuFind/src/VuFind/RecordDriver/MarcBasicTrait.php#L3-L4)